modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -139,6 +139,22 @@ To authenticate using access keys, see <<cloud_storage_access_key>>.
 
 ---
 
+=== cloud_storage_disable_archiver_manager
+
+Use legacy upload mode and do not start archiver_manager.
```

Comment:
I don't know anything about the archiver manager, but guessing this should be `archiver_manager`.

Reply:
I believe it's an upcoming feature.